### ISSUE TITLE
FRC161 Team Fill-In

### ISSUE THREAD

**Issue by MC42** [the-blue-alliance/the-blue-alliance-data#82]
Okay, starting with "Cass Tech is team 161, and they attended nationals" this is from the 1998 nationals team list, taken from 45's history project:

![image](https://user-images.githubusercontent.com/6531081/93142752-a0c48000-f6b4-11ea-83ba-442467058b26.png)

https://drive.google.com/file/d/0B_NPAkx7sAJAT1NWOTNvaGZuTXc/view?usp=sharing (edited) 

Their name was Hurricane XT, and they attended nationals in 1997, sourced from the 1997 Nationals program:

![image](https://user-images.githubusercontent.com/6531081/93142774-ab7f1500-f6b4-11ea-8104-602a4e489814.png)

https://drive.google.com/file/d/0B_NPAkx7sAJANlpNT3BIdmtrTG8/view?usp=sharing (edited) 

Attended GLR in 1998

![image](https://user-images.githubusercontent.com/6531081/93142789-b5a11380-f6b4-11ea-9a99-c849a7ea7d4a.png)

https://drive.google.com/file/d/0B_NPAkx7sAJAbmJOQUg2MkUwYjg/view?usp=sharing

TL;DR?

FRC161 is named "Hurricane TX" and can safely be added to `1998mi` and `1998cmp`.
